src/Microsoft.ML.Transforms/Text/TextFeaturizingEstimator.cs

```diff
@@ -636,46 +643,37 @@ internal static IDataTransform Create(IHostEnvironment env, Options args, IDataV
         private sealed class Transformer : ITransformer
```

Comment:
>Transformer [](start = 29, length = 11)

While we need this class for the backcompat case, would anything bad actually happened if we retained it *only* for back-compat, and just returned chains?

Maybe not -- we might want to expose parts of it later in future API, just an idle thought.